src/api/external-api.js

```diff
@@ -0,0 +1,27 @@
+import {AsgardeoSPAClient} from '@asgardeo/auth-react';
+import endpointConfig from '../configs/endpoint-config';
+
+const externalApiEndpoint = `${endpointConfig.api.endpoints.externalApi}`;
+
+const auth = AsgardeoSPAClient.getInstance();
+
+export async function getExternalApi () {
+
+  const requestConfig = {
+    headers: {
+      "Accept": "application/json",
+      "Content-Type": "application/json"
+    },
+    method: "GET",
+    url: externalApiEndpoint
```

Comment:
Shall we just inline `${endpointConfig.api.endpoints.externalApi}` since the variable isn't used anywhere else.
The variable is taking unnecessary memory.

Reply:
Fixed by a66cbb117ca5bd1bc65d7fff6d176b7a71b932b9.